datasource/index.js

```diff
@@ -48,8 +52,8 @@ module.exports = yeoman.generators.Base.extend({
 
       this.listOfAvailableConnectors = list.map(function(c) {
         var support = c.supportedByStrongLoop ?
-          ' (supported by StrongLoop)' :
-          ' (provided by community)';
+          g.f(' (supported by StrongLoop)') :
+          g.f(' (provided by community)');
         return {
           name: c.description + support,
```

Comment:
A minor globalization issue here is the concatenation `c.description + support`  Text concatenation with `+` operator of multiple strings may compose a right English, but not usually the case in other languages.

This particular case is not a problem.  String concatenation in code is one of the typical globalization bugs.